sdk/inc/azure/core/az_credentials.h

```diff
@@ -125,6 +127,20 @@ AZ_NODISCARD az_result az_credential_client_secret_init(
     az_span client_id,
     az_span client_secret);
 
+/**
+ * @brief sets authentication authority URL for the client secret credential.
+ *
+ * @param ref_credential reference to an az_credential_client_secret instance to set the authority
+ * URL for
+ * @param authority authentication authority URL to set
```

Comment:
nit: Ideally these should be complete sentences (start with capital letter and end with period).

Reply:
Fixed.